docs/mdbook/src/android/map-options-guide.md

```diff
@@ -0,0 +1,308 @@
+# MapLibreMapOptions
+This guide will explain different ways to create a map
+
+When working with maps there are chances of providing default values for MapView to render.
+There are several ways to build MapView:
+1. Providing existing XML namespace tags of MapView in layout
+2. Creating `MapLibreMapOptions` and providing builder function values into MapView
+3. Creating `SupportMapFragment` with the help of `MapLibreMapOptions`
+
+Before explaining MapView configurations we need to know what we can do with both XML namespaces and `MapLibreMaptions`
+
+
+Common configurations can be set:
+- Map base uri
+- Camera
+- Zoom
+- Pitch
+- Gestures
+- Compass
+- Logo
+- Attribution
+- Placement of above elements on the Map and more
+
+We will see how to achieve these configurations in XML layout and programmatically in Activity code step by step
```

Comment:
Each section of the article should have a descriptive heading so people can easily skip to the part they are interested in.

Reply:
There are 3 sections of The Sample app. but  list of common configurations part is not detailed, is that ok to leave them as it is and , adding navigation to 3 sections on the heading section?

Reply:
Sections can be short, that is not a problem. But we should have a heading for each part of the article that covers a different concept.